recommendations/unknown-readmore.md

```diff
@@ -0,0 +1,68 @@
+## Understanding the analysis
+
+An unknown issue occurs when Clinic's analysis algorithms are unable to categorize the sampling results but nevertheless an issue *of some kind* has been detected.
+
+This outcome can be attributed to one of two scenarios:
+ 
+1. Ambient noise – for instance, other applications using the CPU or memory – during the sampling period has polluted the results.
+2. There is a genuine performance issue but Clinic doesn't recognize it.
+
+In the case of ambient noise, there may still be a specific, categorizable performance issue. 
+
+We can make eliminate the possibility of ambient noise and make it easier for Clinic to definitively recognize the issue by:
+
+- Closing down as many applications as possible, especially applications that are CPU- or Memory- intensive. 
+- Using the `--on-port` flag. This can reduce the chances of unknown issues because there is no time gap nor additional system activity between the server starting and the load test beginning.
+
+By way of example, instead of running `clinic -- node app.js` in one terminal and `autocannon localhost:3000` in another, it is preferable and recommended to trigger both in one command using the following command: 
+
+```sh
+clinic --on-port="autocannon localhost:3000" -- node app.js
+``` 
```

Comment:
another simpler alternative is `clinic doctor --autocannon / -- node app.js`. We ship with autocannon now.